src/main/java/frc/robot/subsystems/Auto.java

```diff
@@ -307,13 +307,14 @@ private void Turn(int degrees) {
     }
 
     private void StartGatherer() {
-        shooter.lowerIntake();
-        shooter.startIntake(shooter.intakeSpeed);
+        shooter.gathererState = Shooter.gathererDown;
+        shooter.disableManual = true;
         Done();
     }
 
     private void StopGatherer() {
-        shooter.stopIntake();
+        shooter.gathererState = Shooter.gathererUp;
```

Comment:
this should be the `holding` state, otherwise we will drop the 2nd ball

Reply:
that is only ever called during test init for what we are calling "pit mode". i can explain it tomorrow night if you want

Reply:
I see, I'm confused because it's in Auto.java.

Reply:
your right that has been fixed, although do we ever put the gatherer back up in auto then?

Reply:
I don't think we do